api/src/main/java/org/pmiops/workbench/cohortbuilder/querybuilder/DemoQueryBuilder.java

```diff
@@ -37,16 +40,25 @@ public QueryJobConfiguration buildQueryJobConfig(QueryParameters parameters) {
         List<String> queryParts = new ArrayList<>();
 
         for (SearchParameter parameter : parameters.getParameters()) {
-            final String demoType = parameter.getSubtype().toLowerCase();
-            final String parameterToReplace = "${" + demoType + "}";
-            final String namedParameter = getUniqueNamedParameter(demoType);
-            queryParts.add(getDemoSqlStatement(parameter.getSubtype())
-                    .replace(parameterToReplace, "@" + namedParameter));
-
-            queryParams.put(namedParameter,
-                    parameter.getSubtype().equals("GEN") ?
-                            QueryParameterValue.int64(parameter.getConceptId()) :
-                            QueryParameterValue.int64(new Long(parameter.getValue())));
+            if (parameter.getSubtype().equals("GEN")) {
```

Comment:
Add constants for "GEN", "AGE"?

Reply:
Added an enum: 
public enum DEMOTYPE { GEN, AGE, DEC, RACE };